_projects/2018/astropy/astroquery.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Add additional archives to astroquery
+desc: Implement module(s) to access currently unsupported astronomical archives using astroquery.
+requirements:
+ - Python
+ - Basic understanding of web queries and APIs
+difficulty: low
+issues:
+ - https://github.com/astropy/astroquery/issues/226
+mentors:
+ - keflavich
+ - bsipocz
+ - pllim
+initiatives:
+ - GSOC
+tags:
+ - python
+ - astropquery
+collaborating_projects:
+ - astropy
+ - astroquery
+---
+
+#### Description
+
+[astroquery](http://astroquery.readthedocs.io/), an astropy
+affiliated package, is a package to place modules that access astronomical data
+that is onlone. The aim of this project will be to implement just such modules,
+for a variety of astronomical archives that are currently lacking astroquery modules.
+A list of new services that have been suggested by our users
+[can be found here](https://github.com/astropy/astroquery/issues?q=is%3Aissue+is%3Aopen+label%3A%22New+Service%22) and
```

Comment:
`here` on websites links is [not good practice](https://webaccess.berkeley.edu/ask-pecan/click-here), better to say in the issues of the project.